Opt readme v1 (#3)
AIXjing/juicefs: docs/en/README.md

![JuiceFS LOGO](images/juicefs-logo.png)

> JuiceFS is a high-performance [POSIX](https://en.wikipedia.org/wiki/POSIX) file system released under Apache License 2.0, particularly designed for the cloud-native environment. The data, stored via JuiceFS, itself will be persisted in object storage (e.g. Amazon S3), and the corresponding metadata can be persisted in various database engines such as Redis, MySQL, and SQLite based on the scenarios and requirements.

Reviewer: ```suggestion
JuiceFS is a high-performance [POSIX](https://en.wikipedia.org/wiki/POSIX) file system released under Apache License 2.0, specially optimized for the cloud-native environment. When using JuiceFS to store data, the data itself will be persisted in object storage (e.g. Amazon S3), and the metadata corresponding to the data stored will be persisted in various database engines such as Redis, MySQL, and SQLite based on scenarios and performance requirements.
```

> With JuiceFS, massive cloud storage can directly conntect to big data, machine learning, artificial intelligence, and various application platforms in production environment without modifying code, which can be used as efficiently as local storage. 

Reviewer: ```suggestion
With JuiceFS, massive cloud storage can be directly connected to big data, machine learning, artificial intelligence, and various application platforms in production environments. Without modifying code, the massive cloud storage can be used as efficiently as local storage. 
```

## Highlighted Features

> 1. **Fully POSIX-compatible**: Use like a local file system, seamlessly docking with existing applications without intervening onging work.

Reviewer: ```
1. **Fully POSIX-compatible**: Use as a local file system, seamlessly docking with existing applications without breaking business workflow.
```

> 2. **Fully Hadoop-compatible**: JuiceFS [Hadoop Java SDK](deployment/hadoop_java_sdk.md) is compatible with Hadoop 2.x and Hadoop 3.x as well as a variety of components in the Hadoop ecosystem.

Reviewer: ```suggestion
2. **Fully Hadoop-compatible**: JuiceFS' [Hadoop Java SDK](deployment/hadoop_java_sdk.md) is compatible with Hadoop 2.x and Hadoop 3.x as well as a variety of components in the Hadoop ecosystem.
```

> 3. **S3-compatible**:  JuiceFS [S3 Gateway](deployment/s3_gateway.md) provides S3-compatible interface.

Reviewer: ```suggestion
3. **S3-compatible**:  JuiceFS' [S3 Gateway](deployment/s3_gateway.md) provides an S3-compatible interface.
```

> 1. **Fully POSIX-compatible**: Use like a local file system, seamlessly docking with existing applications without intervening onging work.

Reviewer: ```suggestion
1. **Fully POSIX-compatible**: Use as a local file system, seamlessly docking with existing applications without breaking business workflow.
```

> 4. **Cloud Native**: Easy to use JuiceFS in Kubernetes through [Kubernetes CSI driver](deployment/how_to_use_on_kubernetes.md).

Reviewer: ```suggestion
4. **Cloud Native**:  A [Kubernetes CSI driver](deployment/how_to_use_on_kubernetes.md) is provided for easily using JuiceFS in Kubernetes.
```
